Dockerfile

```
@@ -0,0 +1,22 @@
+FROM ghcr.io/jorgensd/multicompartment-solute-transport:v0.1.0
```

```suggestion
FROM ghcr.io/jorgenriseth/multicompartment-solute-transport:v0.1.0
```
Change base docker image user to @jorgenriseth